Documentation/README_ObjectManipulator.md

```diff
@@ -0,0 +1,242 @@
+# Object manipulator
+
+![Object manipulator](../Documentation/Images/ManipulationHandler/MRTK_Manipulation_Main.png)
+
+The *ObjectManipulator* is the new component for manipulation behaviour, previously found in *ManipulationHandler*. The object manipulator makes a number of improvements and simplifications, and future manipulation features and bug fixes will be made for this component, whereas the manipulation handler will be deprecated.
```

Comment:
"future manipulation features and bug fixes will be made for this component, whereas the manipulation handler will be deprecated." -> "this component is a replacement for ManipulationHandler"

Mainly a nit around the language here, since "future manipulation features and bug fixes will be made for this component" -> can lead to question of "what features and bug fixes" but I think the intent is just that this is the new hotness and don't use the old not hotness